impl_core/src/main/java/io/opencensus/implcore/trace/export/SampledSpanStoreImpl.java

```diff
@@ -276,6 +278,13 @@ public void unregisterSpanNamesForCollection(Collection<String> spanNames) {
   }
 
   @Override
+  public Set<String> listRegisteredSpanNamesForCollection() {
+    synchronized (samples) {
+      return new HashSet<String>(samples.keySet());
```

Comment:
Would it be better to return an immutable set now, to avoid having client code depend on the mutability?

Reply:
Done.